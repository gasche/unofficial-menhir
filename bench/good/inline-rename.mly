%token<int> T
%start<int> a
%%

a: x = c; y = b { x + y }
c: x = T { x }
%inline b : x = c { x }
