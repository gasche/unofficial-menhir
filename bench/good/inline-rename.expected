Warning: you are using the standard library and/or the %inline keyword. We
recommend switching on --infer in order to avoid obscure type error messages.
%start a
%token <int> T

%type <int> a
%%

a:
| x = c x0 = c
    {let y =
  let x = x0 in
  ( x )
in
 x + y }

c:
| x = T
    { x }

%%



